### PR TITLE
Add mTLS Prometheus patching

### PR DESCRIPTION
**Pull request by danielgospodinow**
**Description**

This PR provides Compass with patches to enable mTLS Prometheus scraping.

### CODE REVIEW

File: prow/scripts/cluster-integration/compass-gke-integration.sh [hunk 1]
```diff
@@ -266,6 +266,459 @@ function applyCommonOverrides() {
...
+    sed -i 's/- .\/test-monitoring//g' testdef.yaml
+  fi
+
+  kubectl apply -f testdef.yaml || true
+
+  rm testdef.yaml
+  rm patchSidecarContainerCommand.yaml
+}
+
+function patchCertManager() {
```
[the remark this answers is not in the document]
Comment (danielgospodinow) [reply prow/scripts/cluster-integration/compass-gke-integration.sh:683]: It's not relevant for a local installation, yes. But it's crucial for non-local installations. This shouldn't break local installations that are missing an installed cert manager.

**Review by NickyMateev — APPROVED** [review]

**Pull request merged**